spec/controllers/works/multiple_actions_spec.rb

```diff
@@ -124,6 +124,19 @@
       end
     end
 
+    context "when work parameters are invalid" do
```

Comment:
I'm not sure what the original developer was trying to test, but checking both of those sounds like a good idea to me. I'll have a go.

Reply:
It looks like an unpermitted work parameter actually works fine, which is a minor loophole if people want to be editing their form submissions manually, but not a big security risk since it still goes through the normal work controller checks. And a permitted parameter with an invalid value gives an error in the controller rather than a friendly error page, but that's okay because the form doesn't let you do that unless you edit the form submission manually anyway. So I'm going to scrap this test as it requires coding skills to do either rather than being something a user normally could do.